src/clinical/api/clinical-api.ts

```diff
@@ -94,14 +94,24 @@ class ClinicalController {
 		}
 
 		const data = await service.getClinicalData(programId);
+		const donors = await service.getDonors(programId);
+
+		const donorIds = donors.map((donor: DeepReadonly<Donor>) => donor.donorId);
+		const submitterDonorIds = donors.map((donor: DeepReadonly<Donor>) => donor.submitterId);
+
+		const exceptions =
+			(await getExceptionManifestRecords(programId, {
+				donorIds,
+				submitterDonorIds,
+			})) || [];
 
 		const todaysDate = currentDateFormatted();
 		res
 			.status(200)
 			.contentType('application/zip')
 			.attachment(`${programId}_Clinical_Data_${todaysDate}.zip`);
 
-		const zip = createClinicalZipFile(data);
+		const zip = createClinicalZipFile(data, { programShortName: programId, exceptions });
```

Comment:
Not a problem with this PR, but I looked at the `createClinicalZipFile` function and it never uses the `programShortName` argument, it only needs the list of exceptions.

Reply:
That's odd, surprised VSCode didn't highlight this. I can update this in the next download PR, rather than dismiss your approval.